code/datums/mutantraces.dm

```diff
@@ -2019,6 +2079,25 @@
 			if ("milk")
 				if (mob.emote_check(voluntary))
 					.= release_milk()
+			if ("moo_p")
+				if (mob.emote_check(voluntary, 50))
+					. = list("<B>[mob]</B> moos proudly!", "<i>moos proudly</i>")
```

Comment:
To fit with the cow's insistence that it's totally a cow, maybe this could use a separate message displayed to other cows, or at least to the hummman doing the mooing. Something along the lines of "Human Person makes a proud human noise" or "Human Person grumbles proudly".